### ISSUE TITLE
App doesn’t register any controller inputs 

### ISSUE THREAD

**Issue by kingblah** [moraroy/NonSteamLaunchers-On-Steam-Deck#178]
Tried adding GeForce now but the app doesn’t want to receive any input of any kind. No buttons work, neither does clicking. I made sure the control was set to gamepad with mouse, and tried changing from chrome to edge, and also tried running the konsole override command 

`flatpak —user override —filesystem=/run/udev:ro com.google.Chrome`

and still nothing. I can see the mouse cursor and move it with the trackpad but that’s about it :(

**Comment by moraroy** [comment]
@kingblah you can change the controller settings for any short buy clicking the gamepad button in steamos. you can change it to whatever you want, i gave you the mouse so you can click on things

**Comment by kingblah** [comment]
But clicking doesn’t register either. No buttons do. I can move the cursor; that’s about it. Tried changing the controller settings. Still nothing works

**Comment by moraroy** [comment]
@kingblah that's very strange... does this only happen on the GeForce shortcut or any other one?

**Comment by moraroy** [comment]
@kingblah also you know you can use your finger right for the touch screen? I know ots not the best... bust change the controller to default gamelan and just use touch screen to select a game?

**Comment by kingblah** [comment]
@moraroy Okay so I think it may be a bug with SteamOS / the deck itself! I tried with some other apps - some work (Chiaki4Deck works great) but any browser-based app doesn't work at all. 

In the case of GeForce Now - your suggestion with the touch screen actually worked! While it's in the game selection screen, the touch screen is the only way I can get any input from me to register. I was able to successfully launch a game, and suddenly, the controls work great in the game itself.

I am completely at odds as to what on earth is causing this weird behaviour (I can't even find anyone else experiencing the same issue) but your workaround works great in a pinch.
